interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
One last comment on **search.js**: Considering the number of lines in this file, it may be worth the time to divide it into a few separate files (such as *controller.js*, *services.js*, etc). It is totally up to you. If you want, you can do it in another PR. 

Believe it or not, eslint even allows us to set an upper limit on how many lines a certain *.js file should have. It is disabled in google's eslint rules right now. The default upper limit is 300:
http://eslint.org/docs/rules/max-lines

Everything else in this PR seems good to me. Thanks @rzelayafavila.